templates/include/glean/glean.1.schema.json

```diff
@@ -163,216 +378,196 @@
     "metrics": {
       "properties": {
         "boolean": {
+          "$ref": "#/definitions/boolean"
+        },
+        "labeled_boolean": {
```

Comment:
Oh, TIL "labeled" is American English :)

Reply:
Yeah.  It's annoying.  But I guess Mozilla products should default to American English?